quarkchain/cluster/subscription.py

```diff
@@ -63,9 +63,22 @@ def remove_subscriber(self, sub_id):
 
     async def __notify(self, sub_type, data):
         assert sub_type in self.subscribers
-        for sub_id, websocket in self.subscribers[sub_type].items():
-            response = self.response_encoder(sub_id, data)
-            asyncio.ensure_future(websocket.send(json.dumps(response)))
+        if sub_type == SUB_LOGS:
```

Comment:
the purpose to have `__notify` is to make it generic. how about

1. change the method signature to `async def __notify(self, sub_type, updates)`, then inside the function, iterate `updates` and send them one by one
2. put logic from line 70 to 74 to `notify_log`